test/runtests.jl

```diff
@@ -2,5 +2,5 @@ using QEDevents
 using Test
 
 @testset "QEDevents.jl" begin
-    # Write your tests here.
+    include("interfaces.jl")
```

Comment:
Consider using `@safetestsets` from `SafeTestsets`. For instance, see https://github.com/QEDjl-project/QEDprocesses.jl/blob/dev/test/runtests.jl

Reply:
Implemented in 40475f80c9b50a7d4831a5316723372b2a6c56a9